src/pybind/mgr/volumes/fs/operations/versions/auth_metadata.py

```diff
@@ -0,0 +1,206 @@
+from contextlib import contextmanager
+import os
+import fcntl
+import json
+import logging
+import struct
+import uuid
+
+import cephfs
+
+log = logging.getLogger(__name__)
+
+class AuthMetadataError(Exception):
+    pass
+
+class AuthMetadataManager(object):
+
+    # Current version
+    version = 1
```

Comment:
should this version be 1, or the latest ceph_volume_client version that this module is usurping?